### ISSUE TITLE
Vultr now blocks port 25

### ISSUE THREAD

**Issue by Jocomol** [LukeSmithxyz/emailwiz#313]
Yesterday I asked Vultr to open port 25 but they told me that they don't do this anymore as a security measure. I assume that most of us here are using Vultr and would like to discuss some questions:

- Did Vultr block Port 25 for people who had it unblocked aswell?
- Has anybody experienced the same thing or do they refuse to unblock me specifically (they said they block everyone)
- Are there any other VPS which allow port 25 to be unblocked

Thank you all very much!

**Comment by Jocomol** [comment]
See https://github.com/LukeSmithxyz/emailwiz/issues/172

**Comment by bd-912** [comment]
Hi, was this issue solved for you? They told me the new security measure applies to all 'new accounts', but the port will never be unblocked, regardless of my account age.

Did you find another VPS?

Thanks

**Comment by Jocomol** [comment]
@bd-912 Yes, they actually unlocked me on the grounds that I am a customer of them for three years.

Currently my mailserver runs on exoscale though. They unblocked port 25 for me because I am an arch linux user, not kidding.

**Comment by bd-912** [comment]
It's possible they will unblock the port for me too eventually, but I think most of this "security measure" stuff claiming it cannot be done is meant only to discourage users from opening their ports.

When I asked if they could link where in their ToS or FAQ their new policy is described, they instead linked me their anti-spam policy. They did not explain how my personal email use case counted as spam or unsolicited email.

**Comment by bd-912** [comment]
In case anyone benefits from this (and because I think what I said directly above is wrong), Vultr's support quickly initiated a full refund and account closure when I asked. I only had contact with one employee, and they refused to open the port, (purely due to "more stringent security measures"), and made it clear that they would not be able to open it in the future.

It is probably just best to ask for a refund quickly before the free credit runs out if you are trying to start a new account with them.

**Comment by DLopezJr** [comment]
Can we provide a list of confirmed VPS providers to the README? This will probably be the easiest place for everyone to look.

**Comment by fredrikfischer** [comment]
I can confirm that it is still possible to open port 25 on VULTR. However, I was denied twice. I just repeated that I was only going to use the port for personal  use, low volume, no spam, no marketing, no business intrests and always follow their policy.

I suspect the first two denials came from a bot.  I got the two denials from the same "consultant" but on the third reply another "consultant" finally opened the port.

So my suggestion if to keep trying.

**Comment by yiban-nvxing** [comment]
Good morning. I just made a GitHub account to add that Jocomol is not alone and Vultr still has opening SMTP ports wrapped up in red tape. I couldn't tell whether the responses I was getting were automated or just customer service canned responses, but they kept on denying open ports on the grounds of "not aligning with our Policy," and then when I pointed out that this was false, they changed the story to some vague "security measures" issue. They never actually answered any of my questions or addressed any of my points so I think it might have all been bots.

That being said the script otherwise worked perfectly and was dead simple to use for a complete beginner non-web browser computer use. Thanks Luke.

**Comment by fredrikfischer** [comment]
> Good morning. I just made a GitHub account to add that Jocomol is not alone and Vultr still has opening SMTP ports wrapped up in red tape. I couldn't tell whether the responses I was getting were automated or just customer service canned responses, but they kept on denying open ports on the grounds of "not aligning with our Policy," and then when I pointed out that this was false, they changed the story to some vague "security measures" issue. They never actually answered any of my questions or addressed any of my points so I think it might have all been bots.
> 
> That being said the script otherwise worked perfectly and was dead simple to use for a complete beginner non-web browser computer use. Thanks Luke.

My suggestion would be to try a few more times, and be very clear on your intentions. My final message to them was basically:

- I do not have a company, business interest etc.
- I am not intending to use the port for any type of spam, marketing or similar.
- The volume of sent emails will be low, no more than a normal individual.
- There is not any business conducted from the emails
- I am committed to follow everything you state in your policy (https://www.vultr.com/legal/antispam-policy/)

**Comment by yiban-nvxing** [comment]
Thanks, I hope that succeeds for future readers. I closed my Vultr account by the time I saw this, but not before I did insist a few times with points all along those lines, though after the first two they made it very clear with each additional reply that their decision was final and it was clear that they weren't reading what I was typing. I'm not sure if I'm just getting unlucky or if for some reason I'm "suspicious" (LibreWolf user), but I ended up using Migadu instead of true self-hosting and it's been very satisfactory so far. In the future once I know more about computers and the web, I'll try again for the extra privacy and sovereignty, if bots and spam haven't made it more trouble than it's worth by then.
